scalableminds/webknossos
# Use color picker library to fix problems with Firefox

### URL of deployed dev instance (used for testing):
- https://fixcolorpicker.webknossos.xyz/

### Steps to test:
- create a new annotation
- create a tree and change it's color via right click in the tree tab
- create a segment and change it's color via right click in the segments tab
- ensure that the color editing "feels good" (the popover should close when it loses focus etc)

### Issues:
- fixes #6487

------
